packages/@coorpacademy-components/src/organism/review-no-skills/index.js

```diff
@@ -3,12 +3,25 @@ import {NovaCompositionCoorpacademyEmptyStateHomeRevision as EmptyStateHomeRevis
 import style from './style.css';
 import propTypes from './prop-types';
 
-const ReviewNoSkills = ({titleNoSkills, textNoSkills, iconSkillAriaLabel}) => {
+const ReviewNoSkills = ({
+  titleNoSkills,
+  textNoSkills,
+  iconSkillAriaLabel,
+  directionReverse = false
```

Comment:
Maybe just `reverse`?

Reply:
Wait. I thought it mean reversing a list. But looking at the code it's doing something else.

What it does?